app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,26 @@ def show
     @post = @user.posts.find(params[:id])
     @comments = @post.comments
   end
+
+  def create
+    @post = current_user.posts.new(post_params)
+
+    if @post.save
+      redirect_to user_posts_path(current_user)
+
+    else
+      flash[:alert] = 'Something went wrong!'
+      render 'new'
+    end
+  end
```

Comment:
- [ ] @ritikarawat220 Great work adding a method to add new posts, however when I try to create a new post am getting the error below, this is caused by you having a user_id and author_id columns in your schema as shown below, kindly consider removing user_id, added by the last migration file.
![image](https://github.com/ritikarawat220/Blog-App/assets/61408860/cf64c38f-68bb-4a15-9837-b00670de2ead)

![image](https://github.com/ritikarawat220/Blog-App/assets/61408860/fcc53644-917f-48f5-8971-6648c254942f)